code/game/objects/items/crayons.dm

```diff
@@ -749,6 +749,25 @@
 
 		return
 
+	if(ismob(target) && (target.flags_1 & MOB_PAINTABLE_1))
+		if(actually_paints)
+			target.add_atom_colour(paint_color, WASHABLE_COLOUR_PRIORITY)
+			if(isliving(target.loc))
+				var/mob/living/holder = target.loc
+				if(holder.is_holding(target))
+					holder.update_inv_hands()
+				else
+					holder.regenerate_icons()
```

Comment:
regenerate icons explicity says to not use it. what are you trying to update

Reply:
at one point i copypasted parts from the if is object statement below it that applied to both situations and i totally missed this, my bad